Add simple new map sources to whatsnew (#8209)
sunpy/sunpy: docs/whatsnew/7.0.rst

- pandas>=2.1.0
- aiobotocore>=2.6.0

Polarimeter to Unify the Corona and Heliosphere (PUNCH) map source
==================================================================

> There is now support for PUNCH level 1 products within sunpy.

Reviewer: Are these level 1 products? The sample header they gave us has level 3 in the header.

Author: Woops.

.. image:: 7.0-punch.png
   :alt: A plot from sunpy of a PUNCH level 1 file.

> Additional new map sources

Reviewer 2: Why do we have this heading and also PUNCH above? Feels like we should be giving them all pretty equal billing.

Author: But I will fix.

Author: Fixed.

==========================

The following new map sources have been added to sunpy:

* Global Oscillation Network Group (GONG) magnetogram
* Solar Ultraviolet Imaging Telescope (SUIT)
* SOHO EIT Level 1 files

> Maps now plots with auto-alignment by default

Reviewer: ```suggestion
Maps now plot with auto-alignment by default
```

    <BLANKLINE>
    <BLANKLINE>

Improved return types from the `~sunpy.net.hek.HEKClient`
=========================================================

The `~sunpy.net.hek.HEKClient` now returns `~astropy.units.Quantity`, `~astropy.time.Time`, and `~astropy.coordinates.SkyCoord` objects wherever possible.

> This allows for easier interpretation of results from the Heliophysics Event Knowledgebase, particular when it comes to coordinates and bounding boxes.

Reviewer: ```suggestion
This allows for easier interpretation of results from the Heliophysics Event Knowledgebase, particularly when it comes to coordinates and bounding boxes.
```

> Users should be aware that some redundant columns are now dropped from the results table, for example, those columns which only contained unit information.

Reviewer: ```suggestion
Users should be aware that some redundant columns are now dropped from the results table (e.g. those columns which only contained unit information).
```
